### DISCUSSION TITLE
taro native react连接安卓设备失败

### DISCUSSION THREAD

**Discussion by molu-ke** [NervJS/taro#15443] [Q & A]
3天过去，问题还未解决，求助大神
问题如下：
**error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: gradlew.bat app:installDebug -PreactNativeDevServerPort=8081

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:packageDebug'.
> A failure occurred while executing com.android.build.gradle.tasks.PackageAndroidArtifact$IncrementalSplitterRunnable
   > com.android.ide.common.signing.KeytoolException: Failed to read key androiddebugkey from store "E:\appReact\taro-playground\android\app\debug.keystore": keystore password was incorrect

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4m 14s

    at makeError (E:\appReact\taro-playground\node_modules\execa\index.js:174:9)
    at E:\appReact\taro-playground\node_modules\execa\index.js:278:16
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async runOnAllDevices (E:\appReact\taro-playground\node_modules\@react-native-community\cli-platform-android\build\commands\runAndroid\runOnAllDevices.js:109:5)
    at async Command.handleAction (E:\appReact\taro-playground\node_modules\@react-native-community\cli\build\index.js:142:9)
info Run CLI with --verbose flag for more details.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.**

**Comment by lucasakirahori** [comment]
Did you fix the issue? i'm having the same problem :(
